Loop/Managers/CGM/DexCGMManager.swift

```diff
@@ -100,7 +100,7 @@ final class ShareClientManager: CGMManager {
     var debugDescription: String {
         return [
             "## ShareClientManager",
-            "latestBackfill: \(latestBackfill)",
+            "latestBackfill: \(latestBackfill.debugDescription)",
```

Comment:
It's preferable to call String(reflecting: x) vs calling .debugDescription directly because that works with any instances, including those cases when the runtime will synthesize the string representation for you.